cadence/contracts/NFTCatalog.cdc

```diff
@@ -160,14 +160,22 @@ pub contract NFTCatalog {
         }
     }
 
-    pub fun getCatalog() : {String : NFTCatalogMetadata} {
```

Comment:
At best, I think we can add the new functions and put a comment above this saying not to use it.